public/src/components/channelManagement/helpers/validation.tsx

```diff
@@ -100,11 +100,23 @@ export const createDuplicateValidator = (
 export const CURRENCY_TEMPLATE = '%%CURRENCY_SYMBOL%%';
 export const COUNTRY_NAME_TEMPLATE = '%%COUNTRY_NAME%%';
 export const ARTICLE_COUNT_TEMPLATE = '%%ARTICLE_COUNT%%';
+export const PRICE_DIGISUB_MONTHLY = '%%PRICE_DIGISUB_MONTHLY%%';
+export const PRICE_DIGISUB_ANNUAL = '%%PRICE_DIGISUB_ANNUAL%%';
+export const PRICE_GUARDIANWEEKLY_MONTHLY = '%%PRICE_GUARDIANWEEKLY_MONTHLY%%';
+export const PRICE_GUARDIANWEEKLY_ANNUAL = '%%PRICE_GUARDIANWEEKLY_ANNUAL%%';
 
 const VALID_TEMPLATES = {
```

Comment:
Beyond the scope of this PR, but I think we need a refactor of template rules.
These new templates are currently specific to DOTCOM + banner, but this PR enables them on epic/header.
And really the RTE should only display the allowed templates in the UI (e.g. AMP currently shows the article count template even though it's not allowed).
There is a card for this

Reply:
> but this PR enables them on epic/header

I've removed the `prices` attribute from both header and epic models in SDC which (I hope) will prevent those channels displaying a message should the test include a price template - but I haven't tested this with the recent changes.

For SAC (this PR), I'm thinking maybe we can make the RTE fields more aware of their environment? For instance, when we build the UI for these templates, we only want the menu/buttons to display in Banner forms. Similarly, we want to suppress the `bold` button in the highlighted copy field (as bold styling gets added automatically by SDC).

Reply:
Yes, showing the correct templates for the platform/channel is what I meant. I'm ok with this being done in a separate PR though